nasa-gibs/worldview
# WV-3208: Add NOAA-21/VIIRS fires

## Description

Fixes #WV-3208 .

Add three VIIRS/NOAA-21 Fire and Thermal Anomalies layers.

## How To Test


1. Open with [these URL parameters](http://localhost:3000/?v=-216.3018682183372,-107.66502216693405,165.96132410241322,88.26534646286882&l=Reference_Labels_15m(hidden),Reference_Features_15m(hidden),Coastlines_15m,VIIRS_NOAA21_Thermal_Anomalies_375m_Night,VIIRS_NOAA21_Thermal_Anomalies_375m_Day,VIIRS_NOAA21_Thermal_Anomalies_375m_All,VIIRS_NOAA21_CorrectedReflectance_TrueColor,VIIRS_NOAA20_CorrectedReflectance_TrueColor(hidden),VIIRS_SNPP_CorrectedReflectance_TrueColor(hidden),MODIS_Aqua_CorrectedReflectance_TrueColor(hidden),MODIS_Terra_CorrectedReflectance_TrueColor(hidden)&lg=true&t=2024-08-26-T20%3A00%3A31Z)
2. Ensure fires load on map in Geographic and Arctic projections (not Antarctic); ensure the points are clickable when you zoom in; make sure the layer descriptions read well, ensure that you can take image snapshots of the layers. (Note: currently imagery only goes back to Aug 13, 2024)
3. Verify expected result

@nasa-gibs/worldview
